OasisDEX/oasis-borrow
# 🧿 Ajna earn ui skeleton and slider

# [Ajna earn ui skeleton and slider](https://shortcutLink)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- added ajna earn ui skeleton
- added 3x sliders with different ranges to show how behavior in different scenarios
  
## How to test 🧪
  <Please explain how to test your changes>

- use url for opening ajna position, you should see mocked overview section and form with sliders and input
- handling for input and slider, tx handling etc. are not in the scope of this PR
